Content.Shared/_DV/Weapons/Ranged/Components/GunBipodComponent.cs

```diff
@@ -0,0 +1,66 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._DV.Weapons.Ranged.Components;
+
+/// <summary>
+/// Alters the accuracy and firerate of the gun after a DoAfter, immoblizing the wielder.
+/// </summary>
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class GunBipodComponent : Component
+{
+    [ViewVariables(VVAccess.ReadWrite), DataField("minAngle"), AutoNetworkedField]
+    public Angle MinAngle = Angle.FromDegrees(-43);
+
+    /// <summary>
+    /// Angle bonus applied upon the bipod being used.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("maxAngle"), AutoNetworkedField]
```

Comment:
Redundant attributes
```suggestion
    [DataField, AutoNetworkedField]
    public Angle MinAngle = Angle.FromDegrees(-43);

    /// <summary>
    /// Angle bonus applied upon the bipod being used.
    /// </summary>
    [DataField, AutoNetworkedField]
```

Reply:
So true, Magister